modules/rosa-sts-creating-a-cluster-with-customizations-ocm.adoc

```diff
@@ -23,7 +23,7 @@ link:https://docs.aws.amazon.com/vpc/latest/userguide/vpc-sharing.html[AWS Share
 * You have completed the AWS prerequisites for ROSA with STS.
 * You have available AWS service quotas.
 * You have enabled the ROSA service in the AWS Console.
-* You have installed and configured the latest ROSA CLI (`rosa`) on your installation host.
+* You have installed and configured the latest {product-title} (ROSA) CLI (`rosa`) on your installation host.
```

Comment:
As above, {product-title} is used in the opening paragraph so this can just be ROSA CLI

Reply:
Maybe, "ROSA CLI (`rosa`)" as this is first instance?